src/ch/imedias/rsccfx/view/RsccEnterTokenView.java

```diff
@@ -15,10 +15,9 @@
 public class RsccEnterTokenView extends VBox {
 
   private final Rscc model;
-  private final RsccEnterTokenPresenter presenter;
 
-  Label enterTokenlbl;
-  Label loremIpsumlbl;
+  Label enterTokenlbl;  // FIXME: rename all fields to the proper camel case, i. e. enterTokenLbl
+  Label loremIpsumlbl; // FIXME: give the fields more meaningful names than "loremIpsumLbl" or "exampleLbl"
```

Comment:
@Kennox  give the fields more meaningful names than "loremIpsumLbl" or "exampleLbl"